src/rpc/handlers/LedgerEntry.cpp

```diff
@@ -139,6 +139,14 @@ LedgerEntryHandler::process(LedgerEntryHandler::Input input, Context const& ctx)
             key = ripple::keylet::xChainCreateAccountClaimID(input.bridge->value(), input.createAccountClaimId.value())
                       .key;
         }
+    } else if (input.mptIssuance) {
+        auto const mptIssuanceID = ripple::uint192{std::string_view(*(input.mptIssuance))};
+        key = ripple::keylet::mptIssuance(mptIssuanceID).key;
+    } else if (input.mptoken) {
+        auto const holder = ripple::parseBase58<ripple::AccountID>(input.mptoken->at(JS(account)).as_string().c_str());
```

Comment:
Let's try to avoid `.c_str` if possible. 
Use `boost::json::value_to<std::string>()` if you need a string.

Reply:
And the same for any other places you have `.c_str()` atm. 👍